src/components/smart_objects/src/array_schema_item.cc

```diff
@@ -90,6 +92,31 @@ errors::eType CArraySchemaItem::validate(
   return errors::OK;
 }
 
+bool CArraySchemaItem::filterInvalidEnums(
+    SmartObject& Object,
+    const utils::SemanticVersion& MessageVersion,
+    rpc::ValidationReport* report__) {
+  if (SmartType_Array == Object.getType()) {
+    int index = 0;
+    auto array = Object.asArray();
+    for (auto it = array->begin(); it != array->end();) {
+      std::stringstream strVal;
+      strVal << index;
```

Comment:
We ca use https://en.cppreference.com/w/cpp/string/basic_string/to_string

Reply:
Fixed in https://github.com/smartdevicelink/sdl_core/pull/3385/commits/bee65b5d8fe27964ed09db991e9ecd8c3b7d8d2c